docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -824,16 +868,18 @@ quarkus.oidc.public-key=MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAlivFI8qB4D0y
 smallrye.jwt.sign.key.location=/privateKey.pem
 ----
 
-copy link:{quarkus-blob-url}/integration-tests/oidc-tenancy/src/main/resources/privateKey.pem[privateKey.pem] from the `integration-tests/oidc-tenancy` in the `main` Quarkus repository and use a test code similar to the one in the `Wiremock` section above to generate JWT tokens. You can use your own test keys if preferred.
+To generate JWT tokens, copy `privateKey.pem` from the `integration-tests/oidc-tenancy` in the `main` Quarkus repository and use a test code similar to the one in the preceding <<integration-testing-wiremock>> section.
+You can use your own test keys, if preferred.
 
-This approach provides a more limited coverage compared to the Wiremock approach - for example, the remote communication code is not covered.
+This approach provides limited coverage compared to the WireMock approach.
+For example, the remote communication code is not covered.
 
 [[integration-testing-security-annotation]]
 ==== TestSecurity annotation
 
-You can use `@TestSecurity` and `@OidcSecurity` annotations for testing the `service` application endpoint code which depends on the injected `JsonWebToken` as well as `UserInfo` and `OidcConfigurationMetadata`.
+You can use `@TestSecurity` and `@OidcSecurity` annotations to test the `service` application endpoint code that depends on the injected `JsonWebToken` and `UserInfo` and `OidcConfigurationMetadata`.
 
-Add the following dependency:
+To do so, add the following dependency:
```

Comment:
In at least 1 section above, and one suggestion, it is `First, add ...` so lets either use `First, add` or just `Add`

Reply:
Good point. Updated to "First, add....." for consistency.